samples/event_manager_profiler/prj.conf

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+# Enabling assert
+CONFIG_ASSERT=y
+
+# Logger configuration
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=2
+CONFIG_LOG_BACKEND_UART=y
+CONFIG_LOG_STRDUP_BUF_COUNT=10
+
+# Configuration required by Event Manager
+CONFIG_EVENT_MANAGER=y
+CONFIG_EVENT_MANAGER_PROFILER=y
+CONFIG_EVENT_MANAGER_PROFILER_PROFILE_EVENT_DATA=y
```

Comment:
These 2 are for EM profiler tracer, I would prefer not to put them under this comment.